### PR TITLE
determine and install dependencies with {pak}

### LINKED ISSUE #96: package install in Dockerfile fails because it cannot find package "R"
https://github.com/RMI-PACTA/workflow.data.preparation/blob/c08b8fb5a139716997ae2c0e83a01757d51a9a2d/Dockerfile#L22

This seems to be the same problem discovered [here](https://github.com/RMI-PACTA/workflow.transition.monitor/pull/231#issuecomment-1921786348) and elsewhere. {renv} current dev version has been fixed to not include "R" as a R package dependency, but I suspect a better solution would be to follow the same solution as in https://github.com/RMI-PACTA/workflow.transition.monitor/pull/231, which was to drop using {renv} at all and instead use  `pak::local_deps()` or `pak::local_install_deps()`.

### PR DESCRIPTION
**Pull request by cjyetman**
closes #96

### CODE REVIEW

**Comment by cjyetman** [comment]
This seems to work on my machine. Can either @jdhoffa and/or @AlexAxthelm verify that one their machines?

**Comment by jdhoffa** [comment]
Same as #98 , I am not sure where I can find the relevant data prep input data to be able to test this locally.

**Comment by cjyetman** [comment]
> Same as #98 , I am not sure where I can find the relevant data prep input data to be able to test this locally.

Maybe @AlexAxthelm is better suited to answer this because now all of the input files are stored on Azure, and there's two separate "rawdata" directories, and honestly I'm a bit confused myself what the structure is supposed to be and also find it rather difficult to navigate Azure to get to them. I used input files that I already had on my laptop.

**Comment by jdhoffa** [comment]
> > Same as #98 , I am not sure where I can find the relevant data prep input data to be able to test this locally.
> 
> Maybe @AlexAxthelm is better suited to answer this because now all of the input files are stored on Azure, and there's two separate "rawdata" directories, and honestly I'm a bit confused myself what the structure is supposed to be and also find it rather difficult to navigate Azure to get to them. I used input files that I already had on my laptop.

Sounds good! Once that is resolved, I will review this :-) 
(not trying to block things, just trying to be more prudent in general on our PROD repos)

File: Dockerfile [hunk 1]
```diff
@@ -10,17 +10,15 @@ RUN apt-get update \
     libpng-dev libtiff-dev pandoc git libgit2-dev \
     && rm -rf /var/lib/apt/lists/*
 
-RUN Rscript -e 'install.packages(c("pak", "renv"))'
+RUN Rscript -e 'install.packages("pak")'
 
 COPY . /workflow.data.preparation
```
Comment (jdhoffa) [inline Dockerfile:15]: NIT: I recommend copying in `DESCRIPTION` first, installing the `deps` there, and then copying in the rest of the repository. 

That takes advantage of `Dockerfile` caching and ensuring faster builds.

**Review by jdhoffa — APPROVED** [review]
Built Docker image and everything installs as expected.
LGTM

File: Dockerfile (hunk 1 above)
Comment (cjyetman) [reply Dockerfile:15]: opened #109

**Pull request merged**